apps/Standalone/src/templates/app/TemplatesStandaloneDesigner.tsx

```diff
@@ -1,91 +1,140 @@
 import type { ReactNode } from 'react';
 import { ReactQueryProvider, TemplatesDataProvider } from '@microsoft/logic-apps-designer';
-import { loadToken } from '../../environments/environment';
+import { environment, loadToken } from '../../environments/environment';
 import { DevToolbox } from '../components/DevToolbox';
 import type { RootState } from '../state/Store';
 import { TemplatesDesigner, TemplatesDesignerProvider } from '@microsoft/logic-apps-designer';
 import { useQuery } from '@tanstack/react-query';
 import { useSelector } from 'react-redux';
-// import { useNavigate } from 'react-router-dom';
-// import type { Template, LogicAppsV2 } from '@microsoft/logic-apps-shared';
-// import { saveWorkflowStandard } from '../../designer/app/AzureLogicAppsDesigner/Services/WorkflowAndArtifacts';
-// import type { ParametersData } from '../../designer/app/AzureLogicAppsDesigner/Models/Workflow';
+import { useNavigate } from 'react-router-dom';
+import type { Template, LogicAppsV2 } from '@microsoft/logic-apps-shared';
+import { saveWorkflowStandard } from '../../designer/app/AzureLogicAppsDesigner/Services/WorkflowAndArtifacts';
+import type { ParametersData } from '../../designer/app/AzureLogicAppsDesigner/Models/Workflow';
+import axios from 'axios';
 
 const LoadWhenArmTokenIsLoaded = ({ children }: { children: ReactNode }) => {
   const { isLoading } = useQuery(['armToken'], loadToken);
   return isLoading ? null : <>{children}</>;
 };
 export const TemplatesStandaloneDesigner = () => {
   const theme = useSelector((state: RootState) => state.workflowLoader.theme);
-  const {
-    // appId,
-    isConsumption,
-    workflowName: existingWorkflowName,
-  } = useSelector((state: RootState) => state.workflowLoader);
-  // const navigate = useNavigate();
+  const { appId, isConsumption, workflowName: existingWorkflowName } = useSelector((state: RootState) => state.workflowLoader);
```

Comment:
NIT: I think you can theme from this selector as well, I guess there is no need to access twice to the state